src/components/Search/Search.tsx

```diff
@@ -67,9 +67,13 @@ const StyledInput = styled(Input)`
 export interface SearchProps {
     results?: React.ReactNode[];
     /**
-     * Sets the default value
+     * Sets the value
      */
     value?: string;
+    /**
+     * Function to set the value
+     */
+    setValue?: (value: string) => void;
```

Comment:
IMHO `onInputChange` is a better alternative to `setValue`. Imo the `Search` should not care about what goes on around it. Whether we set value or call an API or whatever, should be left to the user (of the `Search`) to decide, so `onInputChange` seems more appropriate to me. You can use it the same way,
`<Search value={value} onInputChange={setValue} />`

Reply:
I agree with @lloydaf here, `onInputChange` is more in line with other components of the design system and will not add confusion to this component